packages/cli/src/cmds/preview/cpUtils.ts

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// NOTE:
+// DO NOT EDIT this file in function plugin.
```

Comment:
Not function plugin...

Reply:
How about just remove these comments?

Reply:
It depends if you need the copy logic for this file as well

Reply:
I think not. Just removed them for simplicity.